src/main/resources/assets/bloodmagic/lang/de_DE.lang

```diff
@@ -1,181 +1,256 @@
+tooltip.BloodMagic.sigil.seer.currentEssence=Gespeicherte Lebensessenz: %d LP
+tooltip.BloodMagic.sigil.seer.currentAltarCapacity=Maximaler Kapazität: %d LP
+tooltip.BloodMagic.sigil.seer.currentCharge=Ladung: %d
+tooltip.BloodMagic.sigil.phantomBridge.desc=&oLaufen auf dünner Luft...
+tooltip.BloodMagic.sigil.whirlwind.desc=&oBesser keinen Rock tragen
+tooltip.BloodMagic.sigil.enderSeverance.desc=&oBringt Enderman in Notlage
+
+tooltip.BloodMagic.bound.sword.desc=&oErlegen der Schwachen
+tooltip.BloodMagic.bound.pickaxe.desc=&oZerstören von Stein ohne Erbarmen
+tooltip.BloodMagic.bound.axe.desc=&oDemonische Entwälderung
+tooltip.BloodMagic.bound.shovel.desc=&oSchrubbt den Dreck vom Boden
+
+tooltip.BloodMagic.sacrificialDagger.desc=Schon ein Stich in den Finger genügt
+tooltip.BloodMagic.slate.desc=Infundierter Stein in einem Blutaltar
+tooltip.BloodMagic.inscriber.desc=Die Zeichen an der Wand.
+
+tooltip.BloodMagic.pack.selfSacrifice.desc=Dieser Rucksack scheuert richtig...
+tooltip.BloodMagic.pack.sacrifice.desc=Beschreibung
+tooltip.BloodMagic.pack.stored=Stored: %d LP
+
+tooltip.BloodMagic.activationCrystal.weak=Aktiviere niedrig-levelige Rituale
+tooltip.BloodMagic.activationCrystal.awakened=Aktiviere mächtigere Rituale
+tooltip.BloodMagic.activationCrystal.creative=Kreativmodus: Aktiviere jedes Ritual
+
+tooltip.BloodMagic.diviner.currentRitual=Ritual:
+tooltip.BloodMagic.diviner.blankRune=Leere Runen: %d
+tooltip.BloodMagic.diviner.waterRune=Wasserrunen: %d
+tooltip.BloodMagic.diviner.airRune=Luftrunen: %d
+tooltip.BloodMagic.diviner.fireRune=Feuerrunen: %d
+tooltip.BloodMagic.diviner.earthRune=Erdrunen: %d
+tooltip.BloodMagic.diviner.duskRune=Runen der Morgendämmerung: %d
+tooltip.BloodMagic.diviner.dawnRune=Runen der Abenddämmerung: %d
+tooltip.BloodMagic.diviner.totalRune=Runen insgesamt: %d
+tooltip.BloodMagic.diviner.extraInfo=Halte Shift für extra Info
+tooltip.BloodMagic.diviner.currentDirection=Richtung: %s
+
+tooltip.BloodMagic.arcaneAshes=Asche gebraucht für einen alchemischen Kreis
+
+tooltip.BloodMagic.telepositionFocus.coords=Koordinaten: (%d, %d, %d)
+tooltip.BloodMagic.telepositionFocus.dimension=Dimensions-ID: %d
+tooltip.BloodMagic.telepositionFocus.weak=Benutzt, um Blöcke zu verschieben
+tooltip.BloodMagic.telepositionFocus.enhanced=Benutzt, um Blöcke zu verschieben
+tooltip.BloodMagic.telepositionFocus.reinforcedBenutzt, um Blöcke zu verschieben
+tooltip.BloodMagic.telepositionFocus.demonicBenutzt, um Blöcke zu verschieben
+
+# Ritual
+ritual.BloodMagic.testRitual=Testritual
+ritual.BloodMagic.waterRitual=Ritual des vollen Frühlings
+ritual.BloodMagic.lavaRitual=Serenade des Nethers
+ritual.BloodMagic.greenGroveRitual=Ritual des grünen Hains
+ritual.BloodMagic.jumpRitual=Ritual des hohen Sprungs
+ritual.BloodMagic.wellOfSufferingRitual=Quelle des Leidens
+ritual.BloodMagic.featheredKnifeRitual=Ritual der gefederten Klinge
+ritual.BloodMagic.regenerationRitual=Ritual der Regeneration
+
+# Chat
```

Comment:
The ritual of the full spring is currently translated as a season, not a water spring. It would be "Ritual der vollen Quelle" instead